### DISCUSSION TITLE
Hide stroke of a two or more intersection of object.

### DISCUSSION THREAD

**Discussion by Avisikta-Nandan** [fabricjs/fabric.js#9945] [Q&A]
We have options.target.intersectsWithObject(obj) , to check two object is intersecting , how to hide the stroke for the area for the intersection area. Please suggest how to achieve this feature. 
<img width="130" alt="intersection" src="https://github.com/fabricjs/fabric.js/assets/173819367/2c9de952-2351-4a45-8291-fb4680b4bf71">

**Comment by asturur** [comment]
There is nothing like that, you should convert the objects to paths, then find a library that let you merge the path and then display the resulting path.
